### PR TITLE
PSP-9014 : Contact Manager: Link Individual Contact to an Existing Or…

### PR DESCRIPTION
**Pull request by eddherrera**
…ganization (mailing address)

### CODE REVIEW

File: source/frontend/src/features/documents/ComposedDocument.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class BatchUploadResponseModel {
         this.errorMessage = (apiResponse as IApiError).details;
       } else {
         this.isSuccess =
-          apiResponse.uploadResponse?.documentExternalResponse?.status === 'Success' ?? false;
+          apiResponse.uploadResponse?.documentExternalResponse?.status === 'Success' ? true : false;
```
Comment (asanchezr) [inline source/frontend/src/features/documents/ComposedDocument.ts:126]: this change doesn't seem related to the PR - could you please take it out?

File: source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/PropertyDetailsTabView.helpers.tsx [hunk 2]
```diff
@@ -48,8 +48,6 @@ export const defaultPropertyInfo: Partial<ApiGen_Concepts_Property> = {
   tenures: [],
   roadTypes: [],
   dataSourceEffectiveDateOnly: EpochIsoDateTime,
-  isSensitive: false,
-  isProvincialPublicHwy: false,
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/PropertyDetailsTabView.helpers.tsx:52]: are these changes related to the bug fix? not quite sure

**Review by asanchezr — APPROVED** [review]

File: source/frontend/src/features/documents/ComposedDocument.ts (hunk 1 above)
Comment (eddherrera) [reply source/frontend/src/features/documents/ComposedDocument.ts:126]: it's a warning from Vite. since left had operator always returns true.
